src/components/ScholarshipDetailCard.jsx

```diff
@@ -215,7 +215,7 @@ export default function ScholarshipDetailCard({ scholarship, preview }) {
                 label={tag}
                 variant="outlined"
                 color="primary"
-                sx={{ mr: 2, color: '#000' }}
+                sx={{ mr: 2, color: (theme) => theme.palette.text.primary }}
```

Comment:
Per [docs](https://mui.com/system/the-sx-prop/#palette), you should be able to simply do:
```suggestion
                sx={{ mr: 2, color: (theme) => 'text.primary' }}
```